python/pyspark/sql/pandas/types.py

```diff
@@ -699,7 +710,9 @@ def convert_timestamp_ntz(value: Any) -> Any:
         elif isinstance(dt, UserDefinedType):
             udt: UserDefinedType = dt
 
-            conv = _converter(udt.sqlType(), _struct_in_pandas="row") or (lambda x: x)
+            conv = _converter(udt.sqlType(), _struct_in_pandas="row", _ndarray_as_list=False) or (
```

Comment:
`_ndarray_as_list` sounds to be `True` for `UserDefinedType`?

Reply:
Modified, thanks!